teeplot/teeplot.py

```diff
@@ -16,16 +18,15 @@
 
 
 def _is_running_on_ci() -> bool:
-    ci_envs = ['CI', 'TRAVIS', 'GITHUB_ACTIONS', 'GITLAB_CI', 'JENKINS_URL']
```

Comment:
Let’s leave as is (with changes), but yeah this project is super old and crusty and doesn’t any styling enforced, so keep an eye out for unintentional restyling